### PR TITLE
2227/lock login attempts frontend

### PR DESCRIPTION
**Pull request by dominikx96**
# Pull Request Template

## Issue Overview

This PR addresses #1927

- [ ] This change addresses the issue in full
- [x] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

Updates login error alerts for public and partners. This PR covers frontend part and it has been prepared using https://github.com/bloom-housing/bloom/pull/2227

## How Can This Be Tested/Reviewed?

400 code:
title: `Please enter a valid email and password. `
description: `For security reasons, after 5 more failed attempts, you’ll have to wait 30 minutes before trying again.`

429 code:
title: `For secruity reasons, this account has been locked.`
description: `You’ll have to wait 30 minutes since the last failed attempt before trying again. `

1. Try to log in using not valid email -> every time you should see error relevant to 400 error code
2. Log in using a valid email less than 5 times -> you should see an error relevant to 400 error code
3. Log in using a valid email more that / 5 times -> you should see an error relevant to 429 error code

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] I have exported any new pieces added to ui-components
- [x] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

File: sites/partners/pages/sign-in.tsx [hunk 1]
```diff
@@ -32,18 +40,21 @@ const SignIn = () => {
...
         console.error(err)
-        setError("authentication", {
-          type: "manual",
-          message: `${t("authentication.signIn.error")} ${t(
-            "authentication.signIn.errorGenericMessage"
-          )}`,
+
+        setRequestError({
+          title: t("authentication.signIn.error"),
+          content: t("authentication.signIn.errorGenericMessage"),
```
Comment (seanmalbert) [inline sites/partners/pages/sign-in.tsx:57]: @dominikx96 ,
For this piece and the same logic for public, can you create a login error handler in shared-helpers that they both can use?

**Review by seanmalbert — CHANGES_REQUESTED** [review]
@dominikx96 ,
This looks good. See my comment in the code to move some of that logic to shared-helpers.

File: sites/partners/pages/sign-in.tsx (hunk 1 above)
Comment (jaredcwhite) [reply sites/partners/pages/sign-in.tsx:57]: I would add, as much as we can de-dup any common auth screens between the two apps, that would be ideal. 👍

**Review by seanmalbert — APPROVED** [review]
@dominikx96 ,
Thanks for splitting those pieces out! I think we're good to merge this in and any further requests for the frontend, if any, will come from the other PR.

**Pull request merged**